test/test_dht.cpp

```diff
@@ -1069,7 +1069,7 @@ TORRENT_TEST(dht)
 		printf("num_active_buckets: %d\n", tbl.num_active_buckets());
 		TEST_EQUAL(tbl.num_active_buckets(), 6);
    
-#if defined TORRENT_DHT_VERBOSE_LOGGING || defined TORRENT_DEBUG
```

Comment:
oh. I just realized this was in a test. I thought this was in dht_tracker.cpp or node.cpp (which I believe used to have this call as well). so, never mind.